contracts/bonding/BondingManager.sol

```diff
@@ -191,29 +182,11 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
         // Must have a non-zero amount bonded to self
         require(del.delegateAddress == msg.sender && del.bondedAmount > 0);
 
-        t.pendingRewardCut = _rewardCut;
-        t.pendingFeeShare = _feeShare;
+        t.rewardCut = _rewardCut;
```

Comment:
That's correct, good spot. This is however related to #301 so I suggest to fix it up there?

Reply:
We fix this up in #322 (which fixes #301), however right now the 3 BondingManager PRs (#318 #322 #323) all build off of each other, but are opened off of the same brach (`expand-active-transcoder-set`). As a result, each PR includes changes that are not related to the main feature that it aims to implement. This makes reviewing each PR based on the original feature it aims to implement a bit tougher since the feature implementation is mixed in with other changes (reviewing commit by commit is possible, but there are GH workflow patterns that can make this easier by not duplicating commits across PRs - see the link to the blog post at the end of this comment). I chose to only review this PR since I thought it represented the end result of implementing each of the features associated with each of the 3 PRs - this made review a bit easier since I didn't have to try to parse out duplicate changes in each of the individual PRs. 

I'm ok with this for now and I would be fine with using this PR branch as the branch to push fix ups to. But, if you want to separate out the review process into separate PRs then I recommend changing the target branches of the individual PRs to:

`expand-active-transcoder-set` <- `nv/fluent-active-set` <- `nv/remove-pending-params` <- `nv/remove-setActiveTranscoders`

This [blog post](https://graysonkoonce.com/stacked-pull-requests-keeping-github-diffs-small/) provides a good description of this "stacked pull request" pattern which makes reviewing set of sub-features that build off of each other as a part of larger feature easier.

Reply:
fixed in https://github.com/livepeer/protocol/pull/323/commits/edc9d5b1008f7a1d3215b4e0fd9cb8de61f15bf1

Will check out the blog post in a moment !